### ISSUE TITLE
formating Hidden elements on the left side

### ISSUE THREAD

**Issue by Nikoos** [donadigo/appeditor#14]
Hi @donadigo,

This is not an issue, but maybe an enhancement for your great application. When the hidden elements are displayed on the left side, there is no easy way to distinguish which applications are hidden or not.

Maybe changing font style in italics or in other color.

Best regards,

Nikos

**Comment by donadigo** [comment]
Yes, I do agree on that, thanks for your suggestion! I will see if I can include it in the next version.

**Comment by donadigo** [comment]
It looks like it could be easily added but unfortunately Granite's SourceList widget doesn't allow for markup in the item name. I could copy the class from Granite to the source here and fix that, but I'll first submit a patch to Granite itself. This is currently on hold.
